program-runtime/src/loaded_programs.rs

```diff
@@ -382,6 +382,22 @@ impl LoadedProgram {
     }
 
     pub fn to_unloaded(&self) -> Option<Self> {
+        match &self.program {
+            LoadedProgramType::LegacyV0(_)
+            | LoadedProgramType::LegacyV1(_)
+            | LoadedProgramType::Typed(_) => {}
```

Comment:
Put the Some() code here

Reply:
There are two valid cases, so it needs to stay a fall-through to behind the match block.

Reply:
Oh, because of #[cfg(test)], gross. Okay.

But also, if CI was passing before with the assert, it means we don't have any
tests that check the None case?